### PR TITLE
Feed styling

### PR DESCRIPTION
**Pull request by ouchadam**
Some basic feed styling and image loading

Before | After
| --- | ---
|feed-before|feed-after

### CODE REVIEW

File: core/src/main/java/com/ouchadam/loldr/feed/FeedActivity.java [hunk 1]
```diff
@@ -28,7 +28,7 @@
     private static final String ACTION = BuildConfig.APPLICATION_ID + ".FEED";
 
     private static final String EXTRA_SUBREDDIT = "subreddit";
-    private static final String DEFAULT_SUBREDDIT = "news";
+    private static final String DEFAULT_SUBREDDIT = "all";
```
Comment (ataulm) [inline core/src/main/java/com/ouchadam/loldr/feed/FeedActivity.java]: makes sense!

File: ui/src/main/java/com/ouchadam/loldr/feed/PostSummaryViewHolder.java [hunk 2]
```diff
@@ -59,14 +52,27 @@ public void bind(final Ui.PostSummary postSummary) {
...
 
+    private void setScoreAndCommentCount(Ui.PostSummary postSummary) {
+        scoreAndCommentCount.setText(postSummary.getScore() + " points" + " | " + postSummary.getCommentCount() + " comments | " + postSummary.getTime());
+    }
+
+    private void setAuthAndSubreddit(Ui.PostSummary postSummary) {
+        authorAndSubreddit.setText(createSpan(postSummary));
+    }
+
+    private Spannable createSpan(Ui.PostSummary postSummary) {
```
Comment (ataulm) [inline ui/src/main/java/com/ouchadam/loldr/feed/PostSummaryViewHolder.java]: Might want to snag the TextView from our work project - it's got the fix for spans to allow setAllCaps and fonts and shiz which is needed

File: ui/src/main/res/layout/view_feed_post_summary.xml [hunk 3]
```diff
@@ -1,50 +1,77 @@
...
+  android:layout_height="wrap_content">
 
-  <TextView
-    android:id="@+id/feed_post_summary_text_title"
+  <View
     android:layout_width="match_parent"
-    android:layout_height="wrap_content" />
+    android:layout_height="1dp"
+    android:layout_gravity="bottom"
+    android:background="#e5e5e5" />
```
Comment (ataulm) [inline ui/src/main/res/layout/view_feed_post_summary.xml]: :mute:

File: ui/src/main/res/layout/view_feed_post_summary.xml [hunk 4]
```diff
@@ -1,50 +1,77 @@
...
 
-  <TextView
-    android:id="@+id/feed_post_summary_text_author"
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content" />
+      <TextView
+        android:id="@+id/feed_post_summary_text_author_and_subreddit"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:textSize="12dp"
```
Comment (ataulm) [inline ui/src/main/res/layout/view_feed_post_summary.xml]: oh textsize in dps eh

**Comment by ataulm** [comment]
:x: on the textsize in dps!

**Comment by ouchadam** [comment]
<- this guy

**Comment by ataulm** [comment]
I'm gonna make my own reddit app with stripprrs and blackjack and a11y

**Comment by ouchadam** [comment]
do it!

**Pull request merged**